### ISSUE TITLE
How to ignore a specific lib?

### ISSUE THREAD

**Issue by vjpr** [facebook/flow#6256]
In a `flow-typed` dir, I have a broken lib that is installed with `flow-typed`.

Is there are way to prevent that lib from being loaded?

Do negation patterns work in `[libs]`?

**Comment by TrySound** [comment]
Use `[ignore]` to treat module as not existing. This way allows you to provide own types in `flow-typed` directory. Use `[untyped]` to treat module as `any`. This is useful when you need to wait until package types will be fixed.

**Comment by vjpr** [comment]
@TrySound The `flow-typed` dir is generated with `flow-typed-cli` in my own codebase. The types are broken in `flow-typed`.

It seems I have to remove the lib from `flow-typed` to fix it.

**Comment by TrySound** [comment]
@vjpr Yes. `flow-typed` should be maintained by hands and be committed to git. flow-typed can optimise only finding libs.

**Comment by vjpr** [comment]
This is painful. Again, a simple thing, but now I have to modify the tooling to workaround this poor behavior.
